app/src/components/landing/Appsumo/Appsumo.tsx

```diff
@@ -124,53 +115,81 @@ const AppSumoModal: React.FC = () => {
     await batch.commit();
   }, [db, appsumoCodes]);
 
+  const createNewWorkspaceForAppSumo = useCallback(async () => {
+    setIsLoading(true);
+
+    const newTeamName = "Team Workspace";
```

Comment:
Are we keeping the new team's name as `Team Workspace`? In the create team flow, we give default name as `<user>'s Workspace (<domain>)`

Reply:
For appsumo we are keeping `Team Workspace` as default name